src/FakeItEasy.Analyzer/packages.config

```diff
@@ -0,0 +1,12 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<packages>
+  <package id="Microsoft.CodeAnalysis.Analyzers" version="1.1.0" targetFramework="portable45-net45+win8" />
+  <package id="Microsoft.CodeAnalysis.Common" version="1.2.1" targetFramework="portable45-net45+win8" />
+  <package id="Microsoft.CodeAnalysis.CSharp" version="1.2.1" targetFramework="portable45-net45+win8" />
+  <package id="Microsoft.CodeAnalysis.CSharp.Workspaces" version="1.2.1" targetFramework="portable45-net45+win8" />
+  <package id="Microsoft.CodeAnalysis.Workspaces.Common" version="1.2.1" targetFramework="portable45-net45+win8" />
+  <package id="Microsoft.Composition" version="1.0.30" targetFramework="portable45-net45+win8" />
+  <package id="NuGet.CommandLine" version="3.3.0" targetFramework="portable45-net45+win8" />

```

Comment:
This can be removed when we address https://github.com/FakeItEasy/FakeItEasy/pull/640/files#r58458364